request/en.txt

```diff
@@ -0,0 +1,65 @@
+Dear ○○,
+
+This is a staff of a Ruby 25th anniversary event.
+Ruby was born in 24th Feb, 1993, so will be 25th birthday.
+We held a event that celebrating and talking about Ruby's past, now and future, on 24th Feb. 2018, in Tokyo.
+
+We're also make a event on web collecting messages for Ruby from over the world Rubyist.We'd like to ask you to write message for Ruby and Rubyist, though I know how busy you are. We are planning to open the messages web pages on 1st Feb.
```

Comment:
We're also holding an event on the web to collect messages for Ruby from Rubyists around the world.  We know how busy you are, but we'd like to ask you to write a message for Ruby and Rubyists!  We are planning to open the messages web pages on Feb 1st.